pkg/debuginfo/client.go

```diff
@@ -73,7 +73,7 @@ func (c *Client) Upload(ctx context.Context, buildID, hash string, r io.Reader)
 	}
 
 	reader := bufio.NewReader(r)
-	buffer := make([]byte, 1024)
+	buffer := make([]byte, 1024*1024*5)
```

Comment:
How about introducing a human-readable constant rather than a magic number? And also it would be amazing if add a comment about the implications of changing the buffer size?